resources/assets/coffee/react/_components/beatmapset-mapping.coffee

```diff
@@ -20,7 +20,7 @@ el = React.createElement
 
 bn = 'beatmapset-mapping'
 
-@BeatmapsetMapping = ({user, beatmapset}) ->
+@BeatmapsetMapping = ({user, beatmapset, beatmap}) ->
```

Comment:
some beatmaps have different statuses than others within the set (e.g. sets with both Loved and Graveyarded maps)